### DISCUSSION TITLE
Brak subskrypcji SilverPartner

### DISCUSSION THREAD

**Discussion by szympulka** [AzureCommunityPL/group#316] [Pomoc]
Hej,

MS w ramach SilverPartner daje 100$ na subskrypcje w Azure. Mój problem polega na tym, że ktoś to aktywował i nie mam dostępu do subskrypcji (nie widzę jej). Przypisałem swoje konto do subskrypcji (mam global admina) przez PartnerNetwork, ale to nie pomogło. Jak mogę odzyskać dostęp do subskrypcji? Zostaje tylko support?

**Comment by smereczynski** [comment]
A przełączyłeś w ustawieniach AAD opcję, która pozwala Ci widzieć wszystkie subskrypcje w tenancie? (zakładając, że ta subskrypcja została uruchomiona w Twoim tenancie) ;)

**Reply by szympulka** [reply]
Sprawdzałem nawet 2 razy :)

**Reply by smereczynski** [reply]
No to obstawiam, że subskrypcja może nie być w Twoim tenancie :)

**Comment by mgrabarz** [comment]
Masz global admina na subskrypcji, ale nie wiesz w jakim jest tenancie? Nie potrafisz jej znaleźć na Portalu, taki problem?

**Reply by mgrabarz** [reply]
Sypię głowę popiołem. "Global admin" jest na tenancie, a nie sub. Dałem się zamotać ;)))
